cellxgene_schema_cli/cellxgene_schema/convert.py

```diff
@@ -69,7 +69,7 @@ def convert(input_file, output_file):
     match_assays = {
         # 'EFO:0010010': ['cell', 'nucleus'], 
         'EFO:0008720': 'nucleus',
-        'EFO:0008722': 'cell',
+        # 'EFO:0008722': ['cell', 'nucleus'], ,
```

Comment:
Small nit, should we remove the commented out lines. Or maybe explain why they are here?